build/build-image/Dockerfile

```diff
@@ -28,5 +28,9 @@ RUN curl -sSL https://github.com/coreos/etcd/releases/download/v3.1.0/etcd-v3.1.
 
 RUN go get -u github.com/golang/lint/golint
 
+RUN git clone https://github.com/duglin/vlinker.git /vlinker
+
+ENV PATH=$PATH:/vlinker/bin
+
```

Comment:
Non-blocking, but food for thought: Since we're adding another `RUN` directive and another `ENV` directive, I might suggest that we glue each of those, respectively, to existing `RUN` and `ENV` directives. Images with few layers are smaller.

I am LGTM'ing regardless.